content/en/cosign/overview.md

```diff
@@ -15,98 +15,96 @@ features:
+```
 
-You'll need to [install Cosign](/cosign/installation/) first, and you will need access to a container registry.
+The bundle contains signing metadata, including the signature and certificate.  
 
-[ttl.sh](https://ttl.sh) offers free, short-lived (hours), anonymous container image
-hosting if you just want to try out these commands.
+The Cosign command requests a certificate from our certificate authority, Fulcio. Fulcio checks your identity by using an authentication protocol (OpenID Connect) to look at your email address. If your identity is correct, Fulcio grants a short-lived, time-stamped certificate. The certificate is bound to the public key to attest to your identity.  This activity is logged using our transparency and timestamping log, Rekor.
+ 
+Note that you don’t need to use a key to sign.  Currently, you can authenticate with Google, GitHub, or Microsoft, which will associate your identity with a short-lived signing key. For more information, read [Keyless Signatures](https://docs.sigstore.dev/cosign/keyless/).
 
-### 1. Generate a keypair
+For more information about Cosign's additional options and features, run the command:
 
-```shell
-$ cosign generate-key-pair
-Enter password for private key:
-Enter again:
-Private key written to cosign.key
-Public key written to cosign.pub
+```
+cosign sign-blob --help
 ```
 
-### 2. Sign a container and store the signature in the registry
+### Verifying a signed blob
 
-```shell
-$ cosign sign --key cosign.key user/demo
-Enter password for private key:
-Pushing signature to: index.docker.io/user/demo:sha256-87ef60f558bad79beea6425a3b28989f01dd417164150ab3baab98dcbf04def8.sig
-```
+To verify a signed blob, you need to provide three pieces of information:
+* The certificate
+* The signature
+* The identity used in signing
+
+You may be provided with a bundle that includes the certificate and signature.  The blob maintainer should provide the trusted identity.
 
-The `cosign` command above prompts the user to enter the password for the private key.
-The user can manually enter the password, or set an environment variable with `COSIGN_PASSWORD` to use a password automatically.
+The following example verifies the signature on file.txt from user "name@example.com" issued by "accounts@example.com".  It uses a provided bundle "cosign.bundle" that contains the certificate and signature.
+
+```
+$ cosign verify-blob <file> --bundle cosign.bundle --certificate-identity=name@example.com 
+                              --certificate-oidc-issuer=https://accounts.example.com
+```
 
-### 3. Verify a container against a public key
+To verify, Cosign queries the transparency log to compare the public key to what’s in Rekor, and checks the timestamp on the signature against the artifact’s entry in the transparency log. The signature is valid if its timestamp falls within the small window of time that the key pair and certificate issued by OpenID Connect were valid.
```

Comment:
Not: Looks like it’s still OpenID Connect instead of certificate authority

Reply:
I thought I fixed this.  OK, fixed.